centreon/www/include/eventLogs/export/Formatter.php

```diff
@@ -58,6 +58,8 @@ class Formatter
     private string $warning = '';
     private string $critical = '';
     private string $unknown = '';
+    private string $ack = '';
+    private string $notif = '';
```

Comment:
```suggestion
    private string $acknowledgement = '';
    private string $notifification = '';
```

Please avoid shortname into variable name to keep it understandable